packages/firestore/test/integration/api/database.test.ts

```diff
@@ -134,8 +129,10 @@ apiDescribe('Database', (persistence: boolean) => {
         .get({ source: 'cache' })
         .then(doc => expect(doc.exists).to.be.true);
       await readerRef.get({ source: 'cache' }).then(
-        () => fail('Expected cache miss'),
-        err => expect(err.code).to.be.equal(Code.UNAVAILABLE)
+        () => {
+          throw new Error('Expected cache miss');
+        },
+        err => expect(err.code).to.be.equal('unavailable')
```

Comment:
The error codes are not exported internally (https://github.com/firebase/firebase-js-sdk/blob/ae319059e4a083a613a2999c1e61d44c9053a0d7/packages/firestore/index.ts)